packages/common/src/downloads/helpers/file-url-fetchers/fetchExportImageDownloadFileUrl.ts

```diff
@@ -0,0 +1,59 @@
+import { request } from "@esri/arcgis-rest-request";
+import {
+  DownloadOperationStatus,
+  IFetchDownloadFileUrlOptions,
+} from "../types";
+
+/**
+ * @private
```

Comment:
please consider moving any net new functions that we do not intend to expose externally to an `_internal` folder (or module) so that they do not get accidentally re-exported later.

Reply:
Great idea, I'll do that now